plugins/inputs/statsd/statsd.go

```diff
@@ -799,6 +799,12 @@ func parseKeyValue(keyvalue string) (string, string) {
 		val = split[1]
 	} else if len(split) == 1 {
 		val = split[0]
+	} else if len(split) > 2 {
```

Comment:
Can you update/add a test to cover this case? It will help verify it fixes the issue. Thanks!

Reply:
add test case for input plugins(statsd) to test parseKeyValue function